loading.bs

```diff
@@ -331,6 +390,433 @@ Note: Applying the signed exchange's response here has the effect of letting a
+                    string ("") if it is not present.
+                1. Create a <{link}> element |linkElement| which <{link/href}>
+                    attribute is |linkTarget|, and <{link/imagesrcset}>
+                    attribute is |imagesrcset|, and <{link/imagesizes}> is
+                    |imagesizes|.
+                1. Let |selected source| and <var ignore=''>selected pixel
+                    density</var> be the URL and pixel density that results from
+                    [=selecting an image source=] given |linkElement|,
+                    respectively.
+                1. If |selected source| is not null, then set |linkTarget| to
+                    the result of [=URL parser|parsing=] |selected source|, with
+                    a base URL of |navigationParams|'s [=navigation
+                    params/response=]'s [=response/URL=].
+            1. Let |headerIntegrity| be an empty string.
+            1. For each |allowedSxgLink| of |allowedSxgLinks|:
+                1. If the first element isn't the same as |linkTarget|, then
+                    continue.
+                1. Let |innerLinkVariants| be the second item of
+                    |allowedSxgLink|.
+                1. Let |innerLinkVariantKey| be the third item of
+                    |allowedSxgLink|.
+                1. Let |storedExchange| be the result of creating an
+                    [=exchange=] with the URL of |linkTarget| and a new
+                    [=response=].
+                1. If |innerLinkVariants| is not the empty string (""), [=header
+                    list/set=] `` `Variants` ``/|innerLinkVariants| in
+                    |storedExchange|'s [=exchange/response=]'s [=response/header
+                    list=].
+                1. If |innerLinkVariantKey| is not the empty string (""),
+                    [=header list/set=] `` `Variant-Key`
+                    ``/|innerLinkVariantKey| in |storedExchange|'s
+                    [=exchange/response=]'s [=response/header list=].
+                1. Let |requestForMatch| be the result of [=creating a
+                    potential-CORS request=] given |linkTarget|, the empty
+                    string, and [=No CORS=].
+                1. If |requestForMatch| doesn't [=matches the stored exchange=]
+                    |storedExchange|, then continue.
+                1. Set |headerIntegrity| to the fourth item of |allowedSxgLink|.
+            1. Let |alternateSXG| be null.
+            1. For each |sxg| of |navigationParams|'s [=navigation
+                params/prefetched subresource signed exchanges=]:
+                1. If |headerIntegrity| is the same as the value of |sxg|'s
+                    [=exchange/response=]'s [=response/header integrity value=]
+                    encoded as a [[CSP]] <a grammar>hash-source</a>, then set
+                    |alternateSXG| to |sxg|.
+            1. If |headerIntegrity| is not an empty string and |alternateSXG| is
```

Comment:
It wasn't obvious why it was ok for |headerIntegrity| to be an empty string here. I think it's that when it's empty, there was no allowed signed exchange, and so we don't need it to have been prefetched ... in which case maybe it'll be clearer to put the "If |headerIntegrity| is not an empty string" condition around both the for loop and what's left of this step?